src/browser/search.py

```diff
@@ -23,3 +24,13 @@ def search(self, query):
     def get_first_link(self):
         return self.query_result["webPages"]["value"][0]["url"]
 
+class DuckDuckGoSearch:
+    def search(self, query):
+        try:
+            self.query_result = DDGS().text(query, max_results=5)
+            return self.query_result
+        except Exception as err:
+            return err
+
+    def get_first_link(self):
+        return self.query_result[0]["href"]
```

Comment:
**Suggestion**: For the DuckDuckGoSearch class's "search" method, consider adding logging for both successful searches and exceptions. This logging can provide valuable insights into the search functionality's behavior and help with troubleshooting. <br> **Code Suggestion**: 
 ```
 +            logger.info(f"Searching DuckDuckGo for query: {query}")
 +            # Existing try-except block
 +        except Exception as err:
 +            logger.error(f"Error searching DuckDuckGo: {err}")
 ```